libraries/client/client_webhook.py

```diff
@@ -103,7 +103,7 @@ def process_webhook(self):
 
         # Preprocess the files
         preprocess_dir = tempfile.mkdtemp(dir=self.base_temp_dir, prefix='preprocess_')
-        results, preprocessor = do_preprocess(rc, repo_dir, preprocess_dir)
+        results, preprocessor = do_preprocess(rc, repo_dir, preprocess_dir, repo_name)
```

Comment:
Shouldn't need repo_name passed since rc is the container passed around to know this.